doc/arbiter.md

```diff
@@ -7,3 +7,7 @@ ROHD HCL implements a generic `abstract` [`Arbiter`](https://intel.github.io/roh
 The [`PriorityArbiter`](https://intel.github.io/rohd-hcl/rohd_hcl/PriorityArbiter-class.html) is a combinational (stateless) arbiter that always grants to the lowest-indexed request.
 
 [PriorityArbiter Schematic](https://intel.github.io/rohd-hcl/PriorityArbiter.html)
+
+## Round Robin Arbiter
+
+The Roun Robin Arbiter is a sequential (stateful) arbiter which grants requests same ways as Priority Arbiter, but keeps track of last request granted.
```

Comment:
It does more than "keep track", it uses it to try to allocate grants "fairly" between all requestors